File: tests/Generator/resource/swift/complex/Operation.swift
class Operation: Codable {
    var arguments: Dictionary<String, Argument>?
    var authorization: Bool?
    var description: String?
    var method: String?
    var path: String?
    var _return: Response?
    var security: Array<String>?
    var stability: Int?
    var _throws: Array<Response>?

    enum CodingKeys: String, CodingKey {
        case arguments = "arguments"
        case authorization = "authorization"
        case description = "description"
        case method = "method"
        case path = "path"
        case _return = "return"
        case security = "security"
        case stability = "stability"
        case _throws = "throws"
    }
}

File: tests/Generator/resource/rust/complex/operation.rs
use serde::{Serialize, Deserialize};
use argument::Argument;
use response::Response;

#[derive(Serialize, Deserialize)]
pub struct Operation {
    #[serde(rename = "arguments")]
    arguments: Option<std::collections::HashMap<String, Argument>>,

    #[serde(rename = "authorization")]
    authorization: Option<bool>,

    #[serde(rename = "description")]
    description: Option<String>,

    #[serde(rename = "method")]
    method: Option<String>,

    #[serde(rename = "path")]
    path: Option<String>,

    #[serde(rename = "return")]
    _return: Option<Response>,

    #[serde(rename = "security")]
    security: Option<Vec<String>>,

    #[serde(rename = "stability")]
    stability: Option<u64>,

    #[serde(rename = "throws")]
    throws: Option<Vec<Response>>,

}

